fabtests/prov/efa/src/rdm_rnr_read_cq_error.c

```diff
@@ -60,7 +60,12 @@ static int rnr_read_cq_error(void)
 	for (i = 0; i < total_send; i++) {
 		do {
 			ret = fi_send(ep, tx_buf, 32, mr_desc, remote_fi_addr, &tx_ctx);
-			if (ret < 0 && ret != -FI_EAGAIN) {
+			if (ret == -FI_EAGAIN) {
+				fi_cq_read(txcq, NULL, 0);
```

Comment:
Why is this cq read necessary?

Reply:
because this test posts rx + rnr_count number of tx, which is way bigger than the default tx queue (4k), tx pkt pool is exhausted and return EAGAIN